game/commander/tasks/primitive/airassault.py

```diff
@@ -10,9 +10,23 @@
 
 @dataclass
 class PlanAirAssault(PackagePlanningTask[ControlPoint]):
```

Comment:
Unused, right? Just delete it. It's in the git history. It will bitrot if it isn't used.

Reply:
Why remove already done work when we want to implement this later. I do understand that you want to keep it clean but then some one else has to do the work later again. Wasted time

Reply:
Someone else will also have to do the work to keep this code "building" while it's unused. It's in the git history whenever someone needs it.